gymnasium/vector/vector_env.py

```diff
@@ -190,6 +192,7 @@ def step(
             >>> infos
             {}
         """
+        raise NotImplementedError(f"{self.__str__()} step function is not implemented.")
```

Comment:
This does seem helpful to include

Reply:
I'm not sure how the formatter didn't catch this function without a body. Is that normal?

Reply:
It is because the docstring counts as a function component